packages/live-update/Package.swift

```diff
@@ -12,7 +12,7 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", branch: "6.0.0"),
         .package(url: "https://github.com/Alamofire/Alamofire.git", .upToNextMajor(from: "5.9.0")),
-        .package(url: "https://github.com/ZipArchive/ZipArchive.git", .upToNextMajor(from: "2.2.3"))
+        .package(url: "https://github.com/ZipArchive/ZipArchive.git", exact: "2.4.3")
```

Comment:
```suggestion
        .package(url: "https://github.com/ZipArchive/ZipArchive.git", .upToNextMinor(from: "2.4.3"))
```

That should work too, right?

Reply:
Yes should be the same. Allthough they wont release any updates to 2.4.x but anyway. lets use upToNextMinor

Reply:
Not that, but if someone has a different plugin with this dependency, you have a little more "freedom".